.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         args: |
           --verbose
         # Optional: show only new issues if it's a pull request. The default value is `false`.
-        only-new-issues: true
+        only-new-issues: false
```

Comment:
Now that all linting issues are addressed, my thought is that any new ones would be introduced by a PR.

Prefer having this flag set to `false` so we avoid PRs passing and then finding surprises when CI fails on `main` post-merge.